examples/hmac/host/src/main.rs

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::{Arg, Command};
+use methods::{HMAC_ELF, HMAC_ID};
```

Comment:
This needs to be updated as well or the build breaks
```suggestion
use hmac_methods::{HMAC_ELF, HMAC_ID};
```